examples/deployment/kubernetes/trillian-map-deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
           - --source=mapserver:http://localhost:8091/metrics
           - --pod-id=$(POD_NAME)
           - --namespace-id=$(POD_NAMESPACE)
-          - --metrics-resolution=5s
+          - --metrics-resolution=60s
```

Comment:
A more recent version of `prometheus-to-sd` would replace this flag with separate `--scrape-interval` and `--export-interval` flags, which might allow better control.

Reply:
Ah, I'll send another PR to update shortly...